libs/access-control/src/casbin/access.rs

```diff
@@ -80,30 +80,22 @@ impl AccessControl {
     obj: ObjectType<'_>,
     act: ActionVariant<'_>,
   ) -> Result<(), AppError> {
-    if enable_access_control() {
-      let change = AccessControlChange::UpdatePolicy {
-        uid: *uid,
-        oid: obj.object_id().to_string(),
-      };
-      self.enforcer.update_policy(uid, obj, act).await?;
-      let _ = self.change_tx.send(change);
-      Ok(())
-    } else {
-      Ok(())
-    }
+    let change = AccessControlChange::UpdatePolicy {
+      uid: *uid,
+      oid: obj.object_id().to_string(),
+    };
+    self.enforcer.update_policy(uid, obj, act).await?;
+    let _ = self.change_tx.send(change);
+    Ok(())
   }
 
   pub async fn remove_policy(&self, uid: &i64, obj: &ObjectType<'_>) -> Result<(), AppError> {
-    if enable_access_control() {
-      self.enforcer.remove_policy(uid, obj).await?;
-      let _ = self.change_tx.send(AccessControlChange::RemovePolicy {
-        uid: *uid,
-        oid: obj.object_id().to_string(),
-      });
-      Ok(())
-    } else {
-      Ok(())
-    }
+    self.enforcer.remove_policy(uid, obj).await?;
```

Comment:
would it be better if `update_policy` and `remove_policy` return an option value. Then change_tx will send if the change is not `None`